pheweb/conf_utils.py

```diff
@@ -2,7 +2,10 @@
 # This module creates the object `conf`.
 # It also offers some configuration-related utility functions.
 
-from . import utils
+try:
```

Comment:
this you need because you run script inside a package and then it's not loaded as in a package and hence relative imports don't work?
seems hairy but you can add direct test of that case https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time. no biggie though,

Reply:
I import the script from another script outside pheweb so that I can read the pheweb fields.